README.md

```diff
@@ -1,2 +1,40 @@
-# focals-api-quickstart
-Starter project to get developers up and running with the Focals Developer API **THIS IS AN ALPHA RELEASE**
+# Focals Developer API Quickstart
+
+This repository is intended to be used as a quickstart to show what's needed to create an ability for the Focals Developer API for Focals by North.  
+The initial example is provided only in NodeJS and is a barebones implementation - so things such as user management, caching, or a full implementation of features isn't provided.  
+
+## Components
+
+### index.js
+This is the entrypoint of the application.  
+In here we set up reading the config file, initialize the `focals-client-js` package, and set up the [Express](https://expressjs.com/) routes and starts the web server.  
```

Comment:
This sentence is a little weird. Maybe `In here we set up reading the config file, initialize the `focals-client-js` package, set up the [Express](https://expressjs.com/) routes and start the web server.`?